src/Components/DropdownItem/DropdownItem.tsx

```diff
@@ -0,0 +1,41 @@
+import classNames from 'classnames';
+import { Person } from '../../types/Person';
+
+type Props = {
+  person: Person;
+  setSelectedPerson: (person: Person) => void,
+  setQuery: (param: string) => void,
+  setAppliedQuery: (param: string) => void,
+};
+
+export const DropdownItem: React.FC<Props> = ({
+  person,
+  setSelectedPerson,
+  setQuery,
+  setAppliedQuery,
+
+}) => {
+  const handlerOnClick = (p: Person) => {
```

Comment:
instead of pusshing two-levels the same props(DropdownItem, DropdownContent) , just  move this `handlerOnClick` to `Dropdown` component